sycl/test-e2e/Graph/Explicit/host_task2_multiple_roots.cpp

```diff
@@ -0,0 +1,11 @@
+// RUN: %{build} -o %t.out
+// RUN: %{run} %t.out
+// Extra run to check for leaks in Level Zero using UR_L0_LEAKS_DEBUG
+// RUN: %if level_zero %{env SYCL_PI_LEVEL_ZERO_USE_IMMEDIATE_COMMANDLISTS=0 UR_L0_LEAKS_DEBUG=1 %{run} %t.out 2>&1 | FileCheck %s --implicit-check-not=LEAK %}
```

Comment:
Use `%{l0_leak_check}` instead of `UR_L0_LEAKS_DEBUG=1`  or you'll get failures on Windows.

Reply:
The tests have been updated this way.